analyze_issue_metadata.py

```diff
@@ -0,0 +1,186 @@
+import csv
+import json
+import os
+import sys
+
+from datetime import datetime as dt
+
+import github3
+
+import get_issue_metadata as gim
+
+
+def main(state):
+
+    """
+    Command line tool that reads in an input file
+    containing GitHub repository names, calls read_issue_metadata
+    and returns a JSON file of issue data.
+
+    Usage (command line):
+        $ python analyze_issue_metadata.py googlecloudplatform repos.txt
+
+    Args:
+        state: GitHub state of issue ('open', 'closed'
+               or 'all')
+        sys.argv[1]: github owner or organization
+        sys.argv[2]: path to input file containing names of repos belonging to
+                     owner or org in sys.argv[1].  One repo name per line.
+
+    Returns:
+        A JSON of issue metadata
+
+    Raises:
+        N/A
+    """
+
+    org = str(sys.argv[1])
+
+    with open(sys.argv[2]) as f:
+        repos = f.readlines()
+    repos = [x.strip() for x in repos]
+
+    # Authenticate with GitHub using Personal Access Token
+    g = github3.login(token=os.environ['GH_TOKEN'])
+
+    data = gim.get_issue_metadata(g, org, repos, state)
+
+    # Dump the data to a JSON file
+    data_json = json.dumps(data)
+    file_time = dt.now().strftime("%Y%m%d")
+    print 'Dumping data to json file'
+    with open('../output_files/' + file_time + '_raw_veneer_issue_meta.json',
+              'w') as f:
+        data = json.dump(data, f, sort_keys=True, indent=4, separators=(
+             ',', ': '))
+
+    return data_json
+
+
+def analyze_issue_metadata(data):
+
+    p0_labels = ['Priority: P0', 'priority: P0', 'priority: p0', 'P0', 'p0']
```

Comment:
Rather than keep track of all the different ways these can be capitalized, you would have an easier time simply making the comparisons case-insensitive later (when I find it, I will add a comment).

Similarly, you may not need to compare the entire label text. In particular, checking for just `enhancement` or `bug` or `question` is probably pretty safe.